paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -941,6 +941,9 @@ class PaymentSheet internal constructor(
             return if (isDark) colorsDark else colorsLight
         }
 
+        @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+        fun getEmbedded() = embeddedAppearance
```

Comment:
Had to add this helper as Embedded is marked as internal

Reply:
Hmm, this doesn't feel right. Because every property should be internal, not just the roots. I think we need another way to track the current.

Reply:
This whole thing shouldn't be possible. See https://github.com/stripe/stripe-android/pull/9731

I think we want to do something more similar to how we do the rest of the configuration in the playground (PlaygroundSettingDefinition) for building up the configuration, and tracking the state separately.